lib/ExpandableListItem/ExpandableListItem.js

```diff
@@ -289,6 +290,19 @@ module.exports = kind(
 			this.bubble('onRequestScrollIntoView');
```

Comment:
@rajyavardhanp I think above code isn't needed anymore, right? :)

{code}
		var direction = inEvent && inEvent.dir;

		if (this.getOpen() && this.getAutoCollapse() && direction === 'UP') {
			this.setActive(false);
		}
{code}